src/ui/theme.rs

```diff
@@ -140,7 +140,7 @@ pub fn load_system_fonts(mut fonts: FontDefinitions) -> FontDefinitions{
     debug!("Attempting to load sys fonts");
     let mut fontdb = HashMap::new();
 
-    fontdb.insert("chinese_simplified", vec![
+    fontdb.insert("asian_fonts", vec![
```

Comment:
This is not strictly true. If the first match supports all asian fonts, yes. But if you ad one font for Korean and one for Japanese, only the Korean is loaded and you end up with no Japanese support. In that case, you need another set, like in the Arabic case.